index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```

Comment:
no CSS link found on your html file. you must add a CSS file to see your style by using  the link  **<link rel="stylesheet" href="">**

Reply:
This is not an issue if there is no CSS in the project, this could be an issue if the project has an unused existing stylesheet and has classes and id referenced between both documents.